content/200-orm/500-reference/200-prisma-cli-reference.mdx

```diff
@@ -1295,30 +1295,33 @@ datasource db {
-| `--to-schema-datamodel`  | Path to a Prisma schema file, uses the data model for the diff                    |                          |
-| `--to-schema-datasource` | Path to a Prisma schema file, uses the URL in the `datasource` block for the diff |                          |
-| `--to-empty`             | Assume that you the data model you are migrating to is empty                      |                          |
+| Options                  | Description                                                                                           | Notes                    |
+| :----------------------- | :---------------------------------------------------------------------------------------------------- | :----------------------- |
+| `--to-url`               | A data source URL                                                                                     |                          |
+| `--to-migrations`        | Path to the Prisma Migrate migrations directory                                                       | Not supported in MongoDB |
+| `--to-schema-datamodel`  | Path to a Prisma schema file, uses the data model for the diff                                        |                          |
+| `--to-schema-datasource` | Path to a Prisma schema file, uses the URL in the `datasource` block for the diff                     |                          |
+| `--to-empty`             | Assume that you the data model you are migrating to is empty                                          |                          |
+| `--to-local-d1`          | Path to a local D1 instance ([learn more](/orm/overview/databases/cloudflare-d1#migration-workflows)) |                          |
 
 Other options:
 
 | Options                 | Required | Description                                                                                                                                       | Notes                                                           |
 | :---------------------- | :------- | :------------------------------------------------------------------------------------------------------------------------------------------------ | :-------------------------------------------------------------- |
 | `--shadow-database-url` | No       | URL for the shadow database                                                                                                                       | Only required if using `--to-migrations` or `--from-migrations` |
 | `--script`              | No       | Outputs a SQL script instead of the default human-readable summary                                                                                | Not supported in MongoDB                                        |
+| `--output`              | No       | Accepts a path for a file to which the output should be written                                                                                   |                                                                 |
```

Comment:
Inspired by `curl -o`'s help message:

```suggestion
| `--output`              | No       | Writes to a file instead of stdout                                                                                  |                                                                 |
```